src/assistant_library_with_local_commands_demo.py

```diff
@@ -50,6 +50,11 @@ def reboot_pi():
     subprocess.call('sudo reboot', shell=True)
 
 
+def say_ip():
```

Comment:
Please split the addition of this command out into a separate PR. As @divx118 says, it's much better to split changes up - then one can be merged even if the other is still being reviewed.